src/App.js

```diff
@@ -17,31 +17,34 @@ import AdminManagement from './components/AdminManagement';
 import Profile from "./components/Profile"
 import UserProfilePage from './pages/UserProfilePage';
 import { useAuth0 } from '@auth0/auth0-react';
-import axios from 'axios';
-
-const axiosInstance = axios.create({
-  baseURL: "http://localhost:5000",
-  headers: {
-    "Content-Type": "application/json"
-  }
-});
+import { Socket } from './components/ChatService/Socket';
+import ChatPage from './components/ChatService/ChatPage';
 
 export default function App () {
   const {isAuthenticated, isLoading, user} = useAuth0();
-  const [userRole, setUserRole] = useState()
-  const [userObj, setUserObj] = useState()
+  const [userRole, setUserRole] = useState();
+  const [dbUser, setDbUser] = useState();
+  // const [isAuthenticated, setIsAuthenticated] = useState();
 
   useEffect(() => {
     if (!isLoading && isAuthenticated) {
-      axiosInstance.get('/get_user_role', {
-      params: {
-        id: user.sub,
-        email: user.email
-      }
-      }).then(response => {
-        setUserRole(response?.data["role"]);
-        setUserObj(response?.data["user_obj"]);
-      });
+      const requestOptions = {
+        method: 'GET',
+      };
+      fetch(`http://localhost:5000/get_user_role?id=${user.sub}&email=${user.email}`, requestOptions)
```

Comment:
we should touch base about this tomorrow too. I'm confused about the reason for these changes